### ISSUE TITLE
No data sent if data is set to false

### ISSUE THREAD

**Issue by devlato** [traverson/traverson#95]
Hello guys!
I have a problem: the target API has a method to change the status of some item. The API method works with PUT method and takes a new status as boolean body value. In my function that sends the data I have the following piece of code:
```js
  // ... bla bla bla
  traverson
      .from(...)
      .follow(...)
      .withTemplateParameters(...)
      .addRequestOptions(...)
      .convertResponseToObject()
      .put(data, (error, result) => {
        if (error) {
          reject(handleError(error));
        } else {
          resolve(result);
        }
      });
```
When I pass data=true for this method, it works and remote API completes successfully. When I pass data=false, remote API method fails because seems like there is no data sent during request.

**Comment by devlato** [comment]
@basti1302

**Comment by basti1302** [comment]
Thank for the report <3

**Comment by basti1302** [comment]
A fix has landed in release 6.0.3. Could you please retest with this release?

**Comment by basti1302** [comment]
Also, using "guys" to address a group of unknown people (possibly mixed gender) is frowned up these days (it is male-centric language). Consider using something like "Hello folks" or "Hello people" or whatever next time ;-)

**Comment by devlato** [comment]
@basti1302 No, I won't use "hello folks", I think "hello guys" is gender neutral enough.

**Comment by devlato** [comment]
Thanks for fix!

**Comment by basti1302** [comment]
First, guys is not gender neutral, period. Do your reading before claiming stuff.

Second, coming to an open source project, asking for a fix, then continue to call the maintainers names and then claiming that the fix is appreciated requires a very special kind of ignorance. 

Please get off my lawn and never come back.

**Comment by devlato** [comment]
@basti1302 There are two moments here:
1. I appreciate your fix and you as a professional. 
2. I don't agree with your point about phrase "hello guys" and it's gender (un)equality.
I don't see any contradictions in my messages, because I responded accordingly to this two points. I mean, I still think that you're a good professional even if I don't agree with your point about gender unequality of the phrase that we were discussing and I still think that your opinion about it is stupid, but it doesn't make you a stupid person or a poor developer in my mind, and I'm still very appreciated about fix.
So I don't understand why you got offended.
P.S. I talked about this phrase with women, and they also think your point is stupid.
